README.md

```diff
@@ -1,13 +1,21 @@
 # grpc-proto
-This repository contains common protocol definitions for peripheral services
-around gRPC such as health checking, load balancing etc..
+This repository contains the canonical common protocol definitions for
+peripheral services around gRPC such as health checking, load balancing etc..
 
-## Planned Usage
+## Usage
 
-For ease of development, proto files will still be copied to the other gRPC
-repositories (e.g., `grpc/grpc`, `grpc/grpc-go`, etc.). Sanity tests will be
-added to verify that common proto files match the "ground truth" files contained
-here.
+For Bazel users, it can be included directly as an http_repository.
+
+For others it is expected the proto files will be copied to other gRPC
```

Comment:
How about:
> Other gRPC repos, and non-Bazel users are expected to copy the  proto files from this repo as needed. However, copied files should not be modified. Changes should be made to proto files in this repo before being recopied. This prevents ...

Reply:
I didn't take this as-is, but integrated a lot of what it suggested.